llm/gpt-2/gpt2-pipeline.yaml

```
@@ -0,0 +1,129 @@
+name: gpt2-data
+
+envs:
+  BUCKET_NAME: # Fill in your bucket name
```

```suggestion
  BUCKET_NAME: # TODO: Fill in your bucket name
```
to align with other examples?